index.html

```diff
@@ -98,8 +98,8 @@ <h3>Generation Beyond</h3>
 						<div class="inner">
 							<header>
 								<!--h2>Alamya Health is ready to help</h2-->
-								<h3>Contact us</h3>
-								<p>Email: sales (at) alamyahealth.com <br>
+								<h2>Contact Us</h2>
+								<p>Email: sales@alamyahealth.com <br>

```

Comment:
Let's try to avoid spambots by generating the email and phone number via javascript as it is done in the First in Class page.